code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -86,8 +86,7 @@
 	var/obj/machinery/nuclearbomb/existing = locate("syndienuke") in GLOB.nuke_list
 	if(existing)
 		qdel(src)
-		throw EXCEPTION("Attempted to spawn a syndicate nuke while one already exists at [existing.loc.x],[existing.loc.y],[existing.loc.z]")
-		return 0
+		CRASH("Attempted to spawn a syndicate nuke while one already exists at [existing.loc.x],[existing.loc.y],[existing.loc.z]")
```

Comment:
So the technically correct code would be . = INITIALIZE_HINT_QDEL then the CRASH